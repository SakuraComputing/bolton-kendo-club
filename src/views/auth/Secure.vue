<template>
    <div>
        <h1>This page is for Members Only</h1>
    </div>
</template>
