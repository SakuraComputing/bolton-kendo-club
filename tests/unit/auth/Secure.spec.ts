import { shallowMount } from '@vue/test-utils'
import Secure from '@/views/auth/Secure.vue'

describe('Secure.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Secure);
    expect(wrapper).toMatchSnapshot()
  })
})
