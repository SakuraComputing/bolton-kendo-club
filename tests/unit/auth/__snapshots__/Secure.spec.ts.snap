// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secure.vue renders correctly 1`] = `
<div>
  <h1>This page is for Members Only</h1>
</div>
`;
